hijack/locale/ru/LC_MESSAGES/django.po

```diff
@@ -2,38 +2,41 @@ msgid ""
 msgstr ""
 "Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2021-03-14 11:27+0100\n"
-"PO-Revision-Date: 2021-03-14 11:36+0100\n"
-"Last-Translator: Johannes Maron <info@johanneshoppe.com>\n"
+"POT-Creation-Date: 2024-04-29 13:23+0200\n"
+"PO-Revision-Date: 2024-04-29 13:24+0200\n"
+"Last-Translator: Johannes Maron <johannes@maron.family>\n"
 "Language-Team: \n"
 "Language: ru\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: Poedit 2.4.2\n"
+"X-Generator: Poedit 3.4.2\n"
 
-#: contrib/admin/admin.py:35
-msgid "hijack user"
-msgstr "захватить пользователя"
+#: contrib/admin/admin.py:64
+msgid "impersonate user"
+msgstr "выдавать себя за пользователя"
 
-#: contrib/admin/templates/hijack/contrib/admin/button.html:8
-msgid "hijack"
-msgstr "захватить"
+#: contrib/admin/templates/hijack/contrib/admin/button.html:7
+msgctxt "verb"
+msgid "impersonate"
+msgstr "выдавать себя за человека"
 
-#: contrib/admin/templates/hijack/contrib/admin/button.html:11
+#: contrib/admin/templates/hijack/contrib/admin/button.html:9
 #, python-format
-msgid "Hijack %(username)s"
-msgstr "Захватить %(username)s"
+msgid "impersonate %(username)s"
+msgstr "захватить %(username)s"
 
-#: templates/hijack/notification.html:7
+#: templates/hijack/notification.html:8
 #, python-format
 msgid "You are currently working on behalf of <em>%(user)s</em>."
 msgstr "В данный момент вы работаете от имени <em>%(user)s</em>."
 
-#: templates/hijack/notification.html:15
+#: templates/hijack/notification.html:16
+msgctxt "verb"
 msgid "hide"
 msgstr "скрыть"
 
-#: templates/hijack/notification.html:18
-msgid "release"
```

Comment:
Not sure this should be changed ("release" looks more specific here).

Reply:
It seemed more fitting to me, considering the message is: `You are currently working on behalf of {{user}}.` Then it would be `stop working`. Yet again, I am not too happy with it either. Maybe another reviewer as good idea.

Reply:
I think purpose of the stop button will when activated is only made clear because of the surrounding the context provided by the 'you are currently working on behalf of...' message.

I agree it could be better, some suggestions:

- stop takeover
- end takeover

Reply:
How about "stop impersonating"

Reply:
Excellent, I will use this. I will also make the other button more verbose.